edubots_surveys.Rnw

```diff
@@ -1496,124 +1496,151 @@ Our initial intention in the design of these surveys was to probe the opinions
+kind of messaging. Some other teaching staff might not use it, but use some
+extraofficial or opt-in channels such as the others included in the survey; in
+the end, between these official and extra-official channels, most people cover
+their needs to communicate with students. This matches the subjective experience
+of the authors of this paper, in general.
+
+Furthermore, the answers to the survey question regarding COVID highlights the
+significant challenge of altering behavior, even during a major crisis. Very few
+responders actually changed their habits, although comparing women vs. men as
+well as those teaching social sciences vs. the rest of the disciplines, there
+are some significant differences. We do not have a clear understanding of the
+reasons behind this behavior; however, it appears that the perception and
+adoption of technologies in a major crisis may vary based on an individual's
+background and gender.
+
+One of the most interesting questions arising from the analysis of the results,
+and that should require further study, is: Are teachers really aware of the
+possibilities of chatbots in the classroom? The survey only listed a few
+functionalities for the chatbots, and just a few subjects selected the option
+``Others''. This may suggest that most teachers did not really understand at
+this point the actual potential of chatbots, since the responses listed were by
+no means exhaustive. Had the responders been more acquainted with the
+possibilities of this technology, this ``Other'' response would possibly have
+shown up many more times. Another way to look at it is that, unless you provide
+the specific instructions on how to use a tool, the teaching staff is already
+too busy or burnt out to think about innovation, so they think it is not their
+job to think about new possibilities of existing or new technologies.
+
+The reluctance to adopt chatbots in class might have an explanation: We can
+speculate that teaching staff's opinions of chatbots are influenced by negative
+experiences with chatbots used in customer support, which is what most people
+are likely to have experienced. Further study may be needed to confirm this, but
+it is important to keep it in mind when introducing chatbot technologies in the
+classroom.  To potentially change this existing perception, it will be necessary
+to make chatbots as simple and satisfying to use as possible.
+
+Regarding gender grouping, both genders agree in the possible uses of chatbots,
+so it seems not relevant at all. However, when we look at the plot containing
+the sector differentiation, it is interesting to note that university teaching
+staff are more likely to use chatbots for grades and FAQs compared to teaching
+staff involved in other tertiary education institutions. This appears logical,
+as the number of personal interactions with students in certain classes at the
+university is smaller\footnote{Most university classes take place one or twice
```

Comment:
```suggestion
university not as frequent\footnote{Most university classes take place once or twice a
```

Reply:
"number... not as frequent" does not ring well. Plus you're disagreeing with ChatGPT, I'll report you to the Higher AI Authority.